CONTRIBUTING.md

```diff
@@ -32,23 +32,23 @@ Consult the Table of Contents below, and jump to the desired section.
 
 ### Enterprise Edition
 
-If you are a Kong Enterprise customer, contact the Enterprise Support channels
+If you are a Kong Enterprise customer, you can contact the Enterprise Support channels
 by opening an Enterprise support ticket on
 [https://support.konghq.com](https://support.konghq.com/).
 
-If you are experiencing a P1 issue, please call the [24/7 Enterprise Support
+If you are experiencing a P1 issue, kindly call us at the [24/7 Enterprise Support
```

Comment:
Please change this back. The Enterprise Support team isn't necessarily the same as "Us", the authors of this document. They are related but not exactly the same thing.

```suggestion
If you are experiencing a P1 issue, please call at the [24/7 Enterprise Support
```